test/specs/interceptors.spec.js

```diff
@@ -32,7 +32,8 @@ describe('interceptors', function () {
     axios.interceptors.request.use(function () {
       return {
         url: '/bar',
-        method: 'post'
+        method: 'post',
+        Promise: Promise
```

Comment:
I don't know the interceptor API very well, and this change concerns me. If interceptors can set the config and defaults don't apply, does the `Promise` property always need to be specified with this addition? Or, should the absence of the `Promise` config here be protected against somewhere else?